src/main/java/com/fvv/bookstore/view/OrderViewImpl.java

```diff
@@ -94,6 +95,109 @@ public void addOrder() {
 		}
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public void listTotalOrdersMonth() {
+		try {
+			StringBuilder sb = new StringBuilder();
+			Integer monthSearch = Integer.parseInt(JOptionPane.showInputDialog("Insert the number month to search"));
```

Comment:
I think its better a single input where the user inform the month and the year altogether. Example:

"Please insert the date to search in the format MM/YYYY"

then, in the code you can split in the "/" and populate both variables related to month and year

Reply:
ok... I use YearMonth